src/type-registry.ts

```diff
@@ -0,0 +1,22 @@
+import { JSORMBase } from './model'
```

Comment:
If this is the `type` related to the JSONAPI payload, let's call it `jsonapi-type-registry` or something along those lines...the word 'type' gets confusing here.

Reply:
good note.